guides/common/modules/con_project-server-overview.adoc

```diff
@@ -0,0 +1,18 @@
+[id="{ProjectServerID}-Overview_{context}"]
+= {ProjectServerTitle} Overview
+
+The {ProjectServer} is the central component of a {Project} deployment where you plan and manage the content life cycle.
+
+A typical {Project} deployment includes one {ProjectServer} on which you perform the following operations:
+
+* Content life cycle management
+* Configuration of {SmartProxyServers} and managed hosts through GUI, CLI, and API
+* System provisioning
+* Patch management
+* Subscription management
+
+The {ProjectServer} delegates content distribution, host provisioning, and communication to the {SmartProxyServers}.
+The {ProjectServer} itself also includes a default, integrated {SmartProxyServer}.
+
+{ProjectServer} also contains a fine-grained authentication system.
+You can grant {Project} operators permissions to access precisely the parts of the infrastructure for which they are responsible.
```

Comment:
Maybe add a link to https://docs.theforeman.org/nightly/Administering_Project/index-katello.html#Managing_Users_and_Roles_admin